src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  color: $color-text-card;
+
+  &:hover {
+    transform: scale(1.2);
+    transition-duration: 300ms;
```

Comment:
To keep the transition smooth when hovering an element, the `transition` property should be declared **not** inside the `:hover` state, but in the element itself.
You should move this declaration to **.card** to fix this problem.
Also, you can use the `transition` property instead of `transition-duration`